_donut.qmd

```diff
@@ -0,0 +1,15 @@
+## Presentation set up guide
```

Comment:
Do you want the presentation setup guide to be visible at the start of the presentation, or as a separate help page on the main list of presentations?

Can just press 's' to get the speaker view.

Reply:
The idea of the presentation set up guide is to help whoever is going to lead the training. The idea is that with the slides loaded, they can read what is on the screen as instructions to get going.

Reply:
I've realised that for the public facing Website, we do not want a donut slide visible. Perhaps any Guide for presenting using Quarto should be in a separate train the trainer repository.